### ISSUE TITLE
Create additional (or update existing) themes

### ISSUE THREAD

**Issue by amanagr** [zulip/zulip-terminal#6]
Simply add your theme to ui.pallete to create a new theme.
Previously added themes have become obsolete and they need to be updated. The `default` theme could also be improved.

**Comment by DhruvThakker** [comment]
I would like to work on this issue

**Comment by amanagr** [comment]
The mechanism for adding using new themes has been added, we can always create a new set palette for adding a theme.

**Comment by reshusinghhh** [comment]
I would like to work on it too. 
Can anyone plz provide the communication channel for Zulip related queries??

**Comment by rht** [comment]
I suppose adding major modern text editors colorscheme would be a start, e.g. onedark[1], gruvbox[2], or classic solarized.

[1] https://github.com/odedlaz/tmux-onedark-theme (https://github.com/joshdick/onedark.vim in vim)
[2] https://github.com/egel/tmux-gruvbox

**Comment by amanagr** [comment]
Great idea @rht

**Comment by neiljp** [comment]
We now have sufficient themes to be going with; the issue is existing themes being out of date.

I have a branch which makes this clearer for end-users and in testing, to show which themes are good or may be incomplete.

Should we close this and add a new issue for updating existing themes?

**Comment by timabbott** [comment]
Yeah, that seems reasonable.

**Comment by neiljp** [comment]
OK, I'll do that once that branch is merged, so we can refer to it.

**Comment by neiljp** [comment]
With the merging of #251, which enables CI & run-time feedback of theme completeness, I'm closing this as discussed above, and will open another issue for updating/improving existing themes.
